src/modules/dashboard/components/ActionsPage/TransactionHash/TransactionHash.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import TransactionMeta from '../TransactionMeta';
+import TransactionStatus from '../TransactionStatus';
+import Hash from './Hash';
+
+import { getMainClasses } from '~utils/css';
+import { STATUS } from '../types';
+
+import styles from './TransactionHash.css';
+
+const displayName = 'dashboard.ActionsPage.TransactionHash';
+
+interface Props {
+  transactionHash: string;
+  status?: STATUS;
+  showMeta?: boolean;
+}
+
+const TransactionHash = ({
+  transactionHash,
+  status,
+  showMeta = true,
+}: Props) => (
+  <div className={getMainClasses({}, styles, { showStatus: !!status })}>
+    {status && (
+      <div className={styles.status}>
+        <TransactionStatus status={status} />
+      </div>
+    )}
+    <div className={styles.transaction}>
+      <Hash transactionHash={transactionHash} />
+      {showMeta && (
+        <TransactionMeta
+          transactionHash={transactionHash}
+          createdAt={new Date()}
+          status={status === STATUS.Failed ? status : undefined}
```

Comment:
Not sure if i got it, so we want show status only if Status is Failed? 
How TransactionMeta component would look like with status = undefined? 
Wondering if we could just pass status here, and in TransactionMeta handle it
```if status===STATUS.FAILED then display something```

Reply:
This was fixed as well, I wonder why you're seeing it, since it's an outdated version.

Maybe refresh ?